website/src/documentation/io/built-in-google-bigquery.md

```diff
@@ -163,6 +163,29 @@ table that you want to write to, unless you specify a [create
 disposition](#create-disposition) of `CREATE_NEVER`. [Creating a table
 schema](#creating-a-table-schema) covers schemas in more detail.
 
+### Data types
+
+BigQuery supports the following data types: STRING, BYTES, INTEGER, FLOAT,
+NUMERIC, BOOLEAN, TIMESTAMP, DATE, TIME, DATETIME and GEOGRAPHY.
+All possible values are described at [https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types] (https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types).
+BigQueryIO allows you to use all of these data types. The following examples
+shows the correct format for data types used when reading from and writing to
+BigQuery:
+
+```java
+{% github_sample /apache/beam/blob/master/examples/java/src/main/java/org/apache/beam/examples/snippets/Snippets.java tag:BigQueryDataTypes
```

Comment:
This code snippet shows empty in the preview link.

Reply:
I don't see a mistake in the syntax, is it possible that the preview actually looks at the master branch and this explains why the code snippet is empty?

Reply:
Hi @melap - does the code snippet change formatting look good to you? For some reason we don't see it in the preview.